en.md

```diff
@@ -89,11 +89,11 @@ We use several offerings of third-party providers to advertise our app on extern
 
 We use the following conversion tracking services in the app:
 
-- Facebook: [https://www.facebook.com/about/privacy](https://www.facebook.com/about/privacy). Facebook has undertaken to uphold the EU-US Privacy Shield.
+- Facebook Inc., 1601 S California Ave, Palo Alto, California 94304, USA; [http://www.facebook.com/policy.php](http://www.facebook.com/policy.php)
```

Comment:
@philipecosia Any reason why you switched from HTTPS for HTTP?

Reply:
We should definitely change it, but actually if you check the link, it automatically redirects to https.

Reply:
Still, wild non-HTTPs links in prod allow for MITM attacks. We shouldn't be those people that facilitate them

Reply:
I didn't see that I'll fix it. Thanks for spotting